frontend/components/Domain/User/UserInviteDialog.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <BaseDialog
+    v-model="inviteDialog"
+    :title="$tc('profile.get-invite-link')"
+    :icon="$globals.icons.accountPlusOutline" color="primary">
+    <v-container>
+      <v-form class="mt-5">
+        <v-select
+v-if="groups && groups.length" v-model="selectedGroup" :items="groups" item-text="name"
```

Comment:
Can we break these long lines up with line breaks? To stay consistent with other components. e.g.
```js
<v-select
  v-if="groups && groups.length
  v-model="selectedGroup"
  :items="groups"
  item-text="name"
  item-value="id"
  :return-object="false"
  filled
  :label="$tc('group.user-group')"
  :rules="[validators.required]"
/>
```